tcms/report/data.py

```diff
@@ -1352,12 +1329,6 @@ def runs_filter_criteria(self, form):
 
         return result
 
-    @staticmethod
-    def get_related_users(testers_ids):
```

Comment:
I don't see this method nowhere else. You have deleted it without replacing it. Have you tried to execute the report views manually and see how that goes ?

Reply:
This method is used only in `walk_case_runs()` in `report.views`.
When i was merging the two `walk_case_runs` methods into a separate module, i also moved their helper methods with them.
After we decided not to merge them, i reverted the changes, but i guess i forgot about this one method.